packages/dso-toolkit/src/components/card/card.mixins.scss

```diff
@@ -3,46 +3,17 @@
 @use "../../variables/media-query-breakpoints";
 @use "../../components/button";
 
-%card-grid {
-  display: grid;
-  grid-template-columns: 1fr minmax(0, 100%);
-
-  .dso-card-heading,
-  .dso-card-content {
-    grid-column: 2 /-1;
-  }
-}
-
-%card-grid-addon {
-  grid-row: span 2;
-}
+@use "card.variables" as card-variables;
 
 @mixin base() {
```

Comment:
De mixins zijn er alleen om styling tussen de Core & HTML/CSS te centraliseren. Als ik het goed zie, wordt deze mixin nu alleen door HTML/CSS gebruikt. Ofwel, dan kan de mixin worden opgedoekt

Reply:
Overigens geldt dat voor alle mixins in deze file. Ze voegen alleen maar complexiteit toe, die het waard is als we plezier hebben van gecentraliseerde styling.

Maar als het niet nodig is, wordt het alleen maar simpeler.

Overigens is de volgende laag van centraliseren dan variables. Als de implementaties geen gedeelde styling hebben is het wel de bedoeling dat ze verwijzen naar dezelfde Sass variables.

Reply:
Pak jij deze, @hansgrimm ?

Reply:
mixins zijn eruit.